awxkit/awxkit/api/pages/workflow_job_template_nodes.py

```diff
@@ -32,6 +32,12 @@ def payload(self, workflow_job_template, unified_job_template, **kwargs):
             'extra_data',
             'identifier',
             'all_parents_must_converge',
+            # prompt fields for JTs
+            'job_slice_count',
+            'labels',
```

Comment:
I don't think we want labels here, because it's not a direct field, so providing it to the factories won't have the desired effect anyway. Otherwise this all looks great.

Reply:
Fixed.